src/Psl/Collection/MutableVector.php

```diff
@@ -623,6 +623,8 @@ public function slice(int $start, ?int $length = null): MutableVector
      *                                         `MutableVector` split into chunks of the given size.
      *
      * @psalm-mutation-free
+     *
+     * @psalm-suppress LessSpecificImplementedReturnType - I don't see how this one is less specific than its inherited.
```

Comment:
Maybe you know?

Reply:
🤷‍♂️ there's actually more of these even with psalm v4